src/domains/chart/components/lib-charts/alarmBadge.js

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef } from "react"
+import styled from "styled-components"
+
+export const getBackgroundColor = (status) => ({
+  WARNING: "#FFF8E1",
+  CRITICAL: "#FFEBEF",
+  CLEAR: "#E5F5E8",
+})[status] || null
+
+export const getBorderColor = (status) => ({
+  WARNING: "#FFC300",
+  CRITICAL: "#F59B9B",
+  CLEAR: "#68C47D",
+})[status] || null
+
+export const getColor = (status) => ({
+  WARNING: "#536775",
+  CRITICAL: "#FF4136",
+  CLEAR: "#00AB44",
+})[status] || null
+
+const Container = styled.div`
+  position: absolute;
+  margin-right: 10px;
+  overflow: hidden;
+  pointer-events: none;
+  direction: rtl;
+`
+
+const Badge = styled.div`
+  display: inline-block;
+  border-radius: 36px;
+  padding: 2px 12px;
+  background: ${({ background }) => background};
+  border: 1px solid ${({ border }) => border};
+  color: ${({ color }) => color};
+  font-size: 12px;
+  font-weight: 700;
+`
```

Comment:
The Badge could be a composition of

`<Flex as={TextSmall} .../>`

Reply:
Correct me if i'm wrong, but because we use explicit colors, we cannot pass them to `Flex`, right? ie. `border`, `color` and `background`?

Reply:
You are right but even if we are not using all the attributes, it is better to use as many as possible. For instance, 
the `font-size: 12px;` will be filled from the `TextSmall `, the `font-weight: 700;` will be filled by the `strong` attribute, etc.